### PR TITLE
chore(docs):rename 'graphar' as 'incubator-graphar' in some files

### PR DESCRIPTION
**Pull request by yuhaoran1214**
### Reason for this PR

as title

### What changes are included in this PR?

rename 'graphar' as 'incubator-graphar' in some files

### Are these changes tested?

yes

### Are there any user-facing changes?

no

### CODE REVIEW

**Review by SemyonSinchenko — APPROVED** [review]
LGTM!

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
 {
 	"name": "GraphAr",
 	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
-	"image": "ghcr.io/apache/graphar-dev:latest",
```
Comment (acezen) [inline .devcontainer/devcontainer.json:25]: the image name is [graphar-dev](https://github.com/apache/incubator-graphar/pkgs/container/graphar-dev)

File: LICENSE [hunk 2]
```diff
@@ -214,26 +214,26 @@ The text of each license is the standard Apache 2.0 license.
 
 * Apache Spark 3.1.1 and 3.3.4 (https://github.com/apache/spark)
     Files:
-      maven-projects/spark/datasourcs-32/src/main/scala/org/apache/graphar/datasources/GarCommitProtocol.scala
```
Comment (acezen) [inline LICENSE:217]: The files include in the LICENSE are correct, no need to update them.

File: docs/libraries/spark/examples.md [hunk 3]
```diff
@@ -203,8 +203,8 @@ See [GraphAr2Neo4j.scala][graphar2neo4j] for the complete example.
 
 :::
 
-[test-graph-transformer]: https://github.com/apache/incubator-graphar/blob/main/maven-projects/spark/graphar/src/test/scala/org/apache/graphar/TestGraphTransformer.scala
```
Comment (acezen) [inline docs/libraries/spark/examples.md:206]: Ditto, no need to update the links of examples.

File: licenserc.toml [hunk 4]
```diff
@@ -45,19 +45,19 @@ excludes = [
   "cpp/apidoc",
```
Comment (acezen) [inline licenserc.toml:45]: Ditto

File: maven-projects/spark/README.md [hunk 5]
```diff
@@ -73,7 +73,7 @@ The API document is generated in the directory ``spark/graphar/target/site/scala
 ## Running Neo4j to GraphAr example
 
 Spark provides a simple example to convert Neo4j data to GraphAr data.
-The example is located in the directory ``spark/graphar/src/main/scala/org/apache/graphar/example/``.
```
Comment (acezen) [inline maven-projects/spark/README.md:76]: Ditto.

File: maven-projects/spark/README.md [hunk 6]
```diff
@@ -164,7 +164,7 @@ We can write a json configuration file like `import/neo4j.json` to do the import
 Running this example requires `Docker` to be installed, if not, follow [this link](https://docs.docker.com/engine/install/). Run `docker version` to check it.
 
 Spark provides a simple example to convert NebulaGraph data to GraphAr data.
-The example is located in the directory ``spark/graphar/src/main/scala/org/apache/graphar/example/``.
```
Comment (acezen) [inline maven-projects/spark/README.md:167]: Ditto.

File: maven-projects/spark/README.md [hunk 7]
```diff
@@ -237,7 +237,7 @@ The example will import the basketballplayer graph from GraphAr to NebulaGraph a
 ## Running the local LDBC sample data to GraphAr example
 
 we provide a simple example to convert LDBC sample data to GraphAr data.
-this example is located in the directory ``spark/graphar/src/main/scala/org/apache/graphar/example/``.
```
Comment (acezen) [inline maven-projects/spark/README.md:240]: Dotto

**Review by acezen — CHANGES_REQUESTED** [review]
Please double check the changes. Some of them are not necessary.

File: maven-projects/spark/README.md (hunk 6 above)
Comment (yuhaoran1214) [reply maven-projects/spark/README.md:167]: OK，I will fix it

**Comment by acezen** [comment]
actually, I think we do not need to rename the `graphar` to `incubator-graphar`, since the https://github.com/apache/graphar would redirect to https://github.com/apache/incubator-graphar

**Comment by yuhaoran1214** [comment]
> actually, I think we do not need to rename the `graphar` to `incubator-graphar`, since the https://github.com/apache/graphar would redirect to https://github.com/apache/incubator-graphar, we just need to rename that links that can not been redirected.

Oh, I see what you mean, I'm going to recheck and I will rename the links that can not been redirected.

**Pull request closed without merging**